src/config/env.ts

```diff
@@ -0,0 +1,18 @@
+const branch = process.env.NEXT_PUBLIC_BRANCH;
+
+interface Env {
```

Comment:
This should be moved to the `types` folder (`types/env.ts`) and imported into this file using `import type`.